kong/plugins/datadog/handler.lua

```diff
@@ -86,7 +87,10 @@ local function send_entries_to_datadog(conf, messages)
                                 message.service and gsub(message.service.name ~= null and
                                 message.service.name or message.service.host, "%.", "_") or "",
                                 message.response and message.response.status or "-",
-                                consumer_id, metric_config.tags, conf)
+                                consumer_id, metric_config.tags,
+                                conf,
+                                message.route and message.route.name and gsub(message.route.name ~= null and
```

Comment:
I think `message.route.name ~= null` was meant to be outside of the `gsub`, otherwise it could end up doing `gsub(false, ....)`. Also I think this means we're missing a test case

Reply:
I just copied the existing one, I didn't wanna change it. But if we're going to set a default value for the route output, then this is redundant check anyway.